channel-api/src/main/java/com/synopys/integration/alert/channel/api/DistributionEventReceiver.java

```diff
@@ -22,26 +22,53 @@
  */
 package com.synopys.integration.alert.channel.api;
 
+import java.util.Optional;
+
 import com.google.gson.Gson;
+import com.synopsys.integration.alert.common.descriptor.accessor.AuditAccessor;
+import com.synopsys.integration.alert.common.exception.AlertException;
 import com.synopsys.integration.alert.common.persistence.accessor.JobDetailsAccessor;
 import com.synopsys.integration.alert.common.persistence.model.job.details.DistributionJobDetailsModel;
 import com.synopsys.integration.alert.common.workflow.MessageReceiver;
 import com.synopsys.integration.alert.processor.api.distribute.DistributionEventV2;
 
 public abstract class DistributionEventReceiver<D extends DistributionJobDetailsModel> extends MessageReceiver<DistributionEventV2> {
+    private final AuditAccessor auditAccessor;
```

Comment:
Audit handling is tracked in a different internal issue and will be resolved separately.